s27136-NikodemTunowski/zajecia1/instalacja_debian_linux.txt

```diff
@@ -0,0 +1,289 @@
+ro - partycja tylko do odczytu, blokuje zapis/nadpis danych.
+sync - wymuszenie synchronizacji danych z pamieci podrecznej systemu plikow do 
+dysku przed zamontowaniem systemu plikow. gwarantuje integralnosc danych i 
+bezpieczenstwo.
+usrquota - limit zajmowanego miejsca partycji przez danego uzytkownika, 
+zapewnia sprawiedliwy podzial dysku i zapobiega temu ze jakis uzytkownik 
+zajmie miejsce innym.
+grpquota - to samo co usrquota tylko ze limity są stosowane dla grup kont.
+user_xattr - Rzoszerzone atrybuty użytkownika, czyli dodatkowe metadane które 
+są dołączane do plików i folderów, poszerza funkcjonalność systemu plików, 
+pozwala na kompresje plików.
+Reserved blocks - Miejsce na dysku niedostępne dla zwyklego uzytkownika, 
+zapobiega przypadkowemu nadpisaniu lub usunięciu ważnych danych znajdujących 
+się na początku partycji.
+Typical usage - pozwala wybrać sposób uzywania partycji, w zaleznosci co 
+wybiore zmienia sie sposob indeksowania plikow. do wiekszych plikow lepiej 
+zastosowac rzadsze inody.
+Bootable flag - z tej partycji system będzie sie bootowal, nalezy wybrac 
+partycje gdzie znajduje sie folder /boot, inaczejk system sie nie uruchomi.
+Package manager - pomijam, ponieważ nie mam dodatkowych mediow instalacyjnych. 
+mozna tego uzyc jesli ma sie dodatkowe pakiety na innym nosniku.
+Wybieram ftp.task.gda.pl ze względu na najbliższą lokalizacje tego serwera.
+blank HTTP proxy information, ponieważ nie potrzebuje proxy do połączenia z 
+internetem.
+odmawiam w ankiecie najpopularniejszych pakietow ze wzgledu na to ze to 
+instalacja testowa
+w software selection zostawiam tylko Standard system utilities, poniewaz sa to 
+podstawowe pakiety ktore pomagaja w korzystaniu z systemu, nie sa wymagane 
+teoretycznie ale w przypadku potrzeby nie musze doinstalowywac ich recznie.
+Opis Software:
+Debian desktop enviroment - metapakiet zawierajacy wszystkie niezbedne pakiety 
+potrzebne do instalacji srodowisk graficznych.
+GNOME - Nowoczesne i łatwe w użyciu środowisko graficzne z bogatą integracją 
+aplikacji.
+Xfce - Lekki i stabilny pulpit z dużym wyborem aplikacji.
+GNOME Flashback - Dobry wybór dla kogoś, kto preferuje bardziej tradycyjny 
+pulpit i chce lżejszego środowiska graficznego niż zwykly GNOME.
+KDE Plasma - Wysoce konfigurowalny i bogaty w funkcje pulpit.
+Cinnamon - Łączy cechy GNOME 2 i Windows, łatwy w użyciu i konfigurowalny.
+MATE - Bazowany na GNOME 2, mniej zasobożerny system, tradycyjny pulpit.
+LXDE - Bardzo lekki pulpit idealny dla starszych komputerów.
+LXQt - Nowoczesny fork LXDE.
+Web server - instaluje serwer WWW, który pozwoli udostepnic strony internetowe 
+z danej maszyny.
+SSH Server - instaluje serwer SSH, który daje bezpieczny zdalny dostęp do 
+wirtualki. Przydaje się to do administracji systemem, transferu plików i wielu 
+innych.
+Standard system utilities - podstawowe pakiety które mają priorytet "standard"
+instaluje grub-pc ze wzgledu ze debian to jedyny system na tej wirtualce, bez 
```

Comment:
No i znowu. Zlewa się to z instalacją pakietów.
Chociaż jedna interlinia przerwy.

A merytorycznie:
Gdzie można zainstalować bootloader na dysku twardym?
Kiedy należy instalować w którym miejscu?